### PR TITLE
prime-factors: add parameters to exercise placeholder

### LINKED ISSUE #612: prime-factors: add parameters to exercise placeholder
We would like you to help us add expected parameters in placeholder templates as described in #509 to make it easier for learners to start working on the exercise in question.

Feel free to ask any questions, we'll be glad to help!

### PR DESCRIPTION
**Pull request by subkrish**
prime-factors: add parameters to exercise placeholder #612 

Resolves #612

### CODE REVIEW

File: exercises/prime-factors/prime_factors.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
-def prime_factors():
+def prime_factors(number):
```
Comment (m-a-ge) [inline exercises/prime-factors/prime_factors.py]: `natural_number` might be a better feet according to https://github.com/exercism/python/blob/master/exercises/prime-factors/README.md
What do you think?

Comment (subkrish) [reply exercises/prime-factors/prime_factors.py]: @m-a-ge That looks like a better parameter name. I'll make the update immediately.

**Comment by subkrish** [comment]
@m-a-ge Change has been made.

**Comment by m-a-ge** [comment]
Merged, thanks, @subkrish 

Please note that there are some special ['closing issue' keywords](https://help.github.com/articles/closing-issues-using-keywords/) that you can put in description of your PR and Github will take care of closing the corresponding issue automatically after merge. 
It makes our work much easier. I hope this hint will help you to make your future PRs a bit better

**Pull request merged**